releasenotes/notes/0.13/qinfo-states-7f67e2432cf0c12c.yaml

```diff
@@ -0,0 +1,191 @@
+        print('Measure both:', counts)
+
+        # Qubit-0
+        counts0 = psi.sample_counts(shots, [0])
+        print('Measure Qubit-0:', counts0)
+
+        # Qubit-1
+        counts1 = psi.sample_counts(shots, [1])
+        print('Measure Qubit-1:', counts1)
+
+      Return the array of measurement outcomes for each sample
+    
+      .. jupyter-execute::
+
+        from qiskit.quantum_info import Statevector
+
+        psi = Statevector.from_label('-1')
+        shots = 10
+  
+        # Sample memory
+        mem = psi.sample_memory(shots)
+        print('Measure both:', mem)
+
+        # Qubit-0
+        mem0 = psi.sample_memory(shots, [0])
+        print('Measure Qubit-0:', mem0)
+
+        # Qubit-1
+        mem1 = psi.sample_memory(shots, [1])
+        print('Measure Qubit-1:', mem1)
+  - |
+    Adds a :meth:`~qiskit.quantum_info.Statevector.measure` method to the
+    :class:`qiskit.quantum_info.Statevector` and
+    :class:`qiskit.quantum_info.DensityMatrix` quantum state classes. This
+    allows sampling a single measurement outcome from the specified subsystems
+    and collapsing the statevector to the post-measurement computational basis
+    state. For example
+
+    .. jupyter-execute::
+
+      from qiskit.quantum_info import Statevector
+      
+      psi = Statevector.from_label('+1')
+  
+      # Measure both qubits
+      outcome, psi_meas = psi.measure()
+      print("measure([0, 1]) outcome:", outcome, "Post-measurement state:")
+      print(psi_meas)
+
+      # Measure qubit-1 only
+      outcome, psi_meas = psi.measure([1])
+      print("measure([1]) outcome:", outcome, "Post-measurement state:")
+      print(psi_meas)
+  - |
+    Adds a :meth:`~qiskit.quantum_info.Statevector.reset` method to the
+    :class:`qiskit.quantum_info.Statevector` and
+    :class:`qiskit.quantum_info.DensityMatrix` quantum state classes. This
+    allows reseting some or all subsystems to the :math:`|0\rangle` state.
+    For example
+
+    .. jupyter-execute::
+
+      from qiskit.quantum_info import Statevector
+      
+      psi = Statevector.from_label('+1')
+
+      # Reset both qubits
+      psi_reset = psi.reset()
+      print("Post reset state: ")
+      print(psi_reset)
+
+      # Reset qubit-1 only
+      psi_reset = psi.reset([1])
+      print("Post reset([1]) state: ")
+      print(psi_reset)
+upgrade:
+  - |
+    The :class:`qiskit.quantum_info.Statevector` and 
+    :class:`qiskit.quantum_info.DensityMatrix` classes no longer copy the
+    input array if it is already the correct dtype.
+deprecations:
+  - |
+    The ``add``, ``subtract``, and ``multiple`` methods of the
+    :class:`qiskit.quantum_info.Statevector` and 
+    :class:`qiskit.quantum_info.DensityMatrix` classes are deprecated and will
+    be removed in a future release. Instead you shoulde use ``+``, ``-``, ``*``
+    binary operators instead.
+  - |
+    The ``rep`` method of the :class:`qiskit.quantum_info.Statevector` and 
```

Comment:
Is this a typo? Is there a `Statevector.rep` method?

Reply:
It was removed in this release

Reply:
Ok, per @chriseclectic suggestion I just removed this item in: https://github.com/Qiskit/qiskit-terra/pull/4057/commits/6c2791c61f27b03e4aa8a90dc3929e9f5c8612fe